dataplane/saiserver/hostif.go

```diff
@@ -91,6 +91,17 @@ func (hostif *hostif) CreateHostif(ctx context.Context, req *saipb.CreateHostifR
 		PortId:        id,
```

Comment:
can you add `Op: pktiopb.PortOperation_PORT_OPERATION_CREATE` to this struct. eventually we'll remove the Create field